tests/golden_tests/compile_file/tagged_dup.hvm

```diff
@@ -0,0 +1,5 @@
+main =
+  dup #i a b = @x x;
```

Comment:
If I use the same tag across different functions, will they all generate the same dup label in the end?
And if I use separate tags in different functions, will they generate different dup labels?

Reply:
In the moment that I use a tagged dup dup #t a b = 0; ..., every occurrence of #t in the program will have the same u8 value